packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -1896,6 +1888,15 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
       return;
     }
 
+    // If we are not allowing freeform and
+    // allowing autoComplete, handle the input here
+    // since we have marked the input as readonly
+
+    if (!allowFreeform && autoComplete === 'on') {
+      this._onInputChange(ev.key);
```

Comment:
I'm not sure that ev.key can just be used like this. It looks like ev.key will map to things like `Enter`, `Escape` and other "named" keys like `F1`

See the console in this [codepen](https://codepen.io/slimemold/pen/zYxEPdX?editors=1111) for a demonstration, enter is the most obvious one. Also more info here on [mdn](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key)

Reply:
Here's the challenge. Take a look at these results using a Russian alphabet keyboard

![image](https://user-images.githubusercontent.com/1434956/71632837-7afa5800-2bc5-11ea-9c4f-d8c51cb0546a.png)

Reply:
Also, the only valid keys when `!allowFreeform && autoComplete` are characters in the list items. Enter still enters, escape still escapes.